### PR TITLE
Add topic creation API

### PR DESCRIPTION
**Pull request by webmakersteve**

### CODE REVIEW

File: lib/admin.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+ *
+ * <code>
+ * var client = AdminClient.create({ ... });
+ * </code>
+ *
+ * Once you instantiate this object, it will have a handle to the kafka broker.
+ * Unlike the other node-rdkafka classes, this class does not ensure that
+ * the upstream broker is connected to. Instead, making an action will validate that.
+ *
+ * @param {object} conf - Key value pairs to configure the consumer
```
Comment (alexander-alvarez) [inline lib/admin.js]: *Admin client

File: lib/admin.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+ */
+AdminClient.prototype.disconnect = function() {
+  LibrdKafkaError.wrap(this._client.disconnect(), true);
+  this._isConnected = false;
+};
+
+/**
+ * Create a topic with a given config.
+ *
+ * @param {NewTopic} topic - Topic partitions with offsets to store offsets for.
```
Comment (alexander-alvarez) [inline lib/admin.js]: This is unclear

File: lib/admin.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+ * This class should not be made using the constructor, but instead
+ * should be made using the factory method.
+ *
+ * <code>
+ * var client = AdminClient.create({ ... });
+ * </code>
+ *
+ * Once you instantiate this object, it will have a handle to the kafka broker.
+ * Unlike the other node-rdkafka classes, this class does not ensure that
+ * the upstream broker is connected to. Instead, making an action will validate that.
```
Comment (alexander-alvarez) [inline lib/admin.js]: `this class does not ensure that it is connected to the upstream broker`

**Comment by Celthi** [comment]
Hi all, Is there any document elaborating how to use to create topic and how to set the replica factor of topic? I cannot find any in the readme file.

**Pull request merged**